contracts/CapabilityFactory.cdc

```diff
@@ -13,45 +13,45 @@
 /// Capabilities is critical to the use case of Hybrid Custody. It's advised to use Factories sparingly and only for
 /// cases where Capabilities must be castable by the caller.
 ///
-pub contract CapabilityFactory {
+access(all) contract CapabilityFactory {
     
-    pub let StoragePath: StoragePath
-    pub let PrivatePath: PrivatePath
-    pub let PublicPath: PublicPath
+    access(all) let StoragePath: StoragePath
+    access(all) let PublicPath: PublicPath
     
```

Comment:
can't delete `PrivatePath` probably

Reply:
I thought deleting fields was valid, but I'm guessing we will find out if this will work once I stage for migration

Reply:
ah yeah you are right, I mixed